python/version_details/version_details.py

```diff
@@ -304,11 +304,18 @@ def destroy(self):
     @property
     def current_entity(self):
         """
-        The current Shotgun entity that is active in the widget.
+        The current Shotgun entity that has been loaded and is active in the widget.
         """
         return self._current_entity
 
     @property
+    def active_entity(self):
```

Comment:
I'm wondering if having these two separate functions is too implementation specific. Is the fact that an entity was requested but has not finished loading from the web mean we should have two separate functions to retrieve the entity?   Should there just be a single `entity` function that behaves like the new `active_entity` instead?

Does the client of this code need to know if the entity is fully loaded? If so we could add a `is_entity_loaded` function that checks if `current_entity` is set.